pyproject.toml

```
@@ -178,6 +180,7 @@ minimal-client = [
     "appdirs",
     "entrypoints",
     "httpx >=0.20.0,!=0.23.1",
+    "json_merge_patch",
```

Kudos for catching this one! I think we also need `jsonpatch` here.

```suggestion
    "json_merge_patch",
    "jsonpatch",
```